src/SoftekBarcodeAPI.php

```diff
@@ -100,9 +106,12 @@ private function extractScanResults(int $resultCount): array
             $bottom = FFI::new($ffiType);
             $right = FFI::new($ffiType);
 
-            $this->ffi->mtGetBarStringPos($this->instance, $barcodeIndex, FFI::addr($left), FFI::addr($top), FFI::addr($right), FFI::addr($bottom));
+            // we pass pointer that we create with the FFI::addr, please ignore the pointer passed here at
+            // this point it is not comparable to php
+            $this->getBarcodePosition($barcodeIndex, FFI::addr($left), FFI::addr($top), FFI::addr($right), FFI::addr($bottom));
 
-            // set referenced types
+            // set referenced types and access on the position variables, the variables are declared
+            // within the barcode tool kit library
```

Comment:
You will have to explain these comments to me. I can't understand what they mean.